src/backends/interflop-mca-mpfr/interflop_mca_mpfr.c

```diff
@@ -329,7 +342,17 @@ static error_t parse_opt(int key, char *arg, struct argp_state *state) {
       } else if (strcasecmp(MCAMODE[MCAMODE_RR], arg) == 0) {
         _set_mca_mode(MCAMODE_RR);
       } else {
-        errx(1, "interflop_mca: --mode invalid value provided, must be one of: {ieee, mca, pb, rr}.");
+        errx(1, "interflop_mca_mpfr: --mode invalid value provided, must be one of: {ieee, mca, pb, rr}.");
+      }
+      break;
+    case 's':
+      errno = 0;
+      int seed = strtol(arg, &endptr, 10);
```

Comment:
no need for the temporary seed. you can directly write inside `ctx-> seed` here and make the code a bit shorter.